### ISSUE TITLE
[Issue] No clear that there are more templates if you scroll when you run `azd up/init`

### ISSUE THREAD

**Issue by jongio** [Azure/azure-dev#66]
The templates appear when I do `azd template list`, but when running `azd up/init` the list is longer than what is visible.  Some are "below the fold".

![image](https://user-images.githubusercontent.com/2163001/178371164-c04d251f-1304-40ca-9915-3cea87d54e4f.png)

It would be great if I could see a visual indication that there are more templates available.

**Comment by ellismg** [comment]
In some of our other lists we prepend a number so this would look like:

```
Empty Template
01. Azure-Samples/todo-nodejs-mongo
02. Azure-Samples/todo-python-mongo
<more templates here>
```

As a sort of way to denote that this was a list and there could be more options.  We could consider doing that here.  We could also expand the number of options shown by default by customizing the `PageSize` property on the select object.

Alternately, we could look into improving survey in some way to show some sort of up and down arrows when your viewport doesn't contain all the entries in the list, but that does seem like maybe more effort than it will be work.

**Comment by vhvb1989** [comment]
I think we should follow the same approach we do for selecting Azure location

- It initially shows the current cursor highlighting the first option:
![image](https://user-images.githubusercontent.com/24213737/178409389-1666eeda-96c8-4674-9cc4-24d2ce1ca3b6.png)

- When using arrows to go down, the cursor goes to the middle and then it starts scrolling down instead of moving beyond the middle:
![image](https://user-images.githubusercontent.com/24213737/178409527-28fbbcdf-9f04-43e9-acc4-ff70d4f392a9.png)

![image](https://user-images.githubusercontent.com/24213737/178409542-408b297a-b081-4e98-8d84-2f5d191dca49.png)

* Note how the cursor stays in the middle and instead of going down, it just scrolls

Once the end of the pages is reached, the scrolling stops:

![image](https://user-images.githubusercontent.com/24213737/178409634-456db54d-c582-447a-86b8-1901159cfb76.png)

**Comment by jongio** [comment]
Can you please research if there is a way to show that there are more items?  Like a down arrow indication.  Thanks

**Comment by vhvb1989** [comment]
Here is a prototype with an option:

https://github.com/Azure/azure-dev/pull/91

It would require to either:
- Propose the change as PR to the 3rd party lib owner
- Create and support our own fork (externally like the prototype)
- Add a copy of the 3rd party src code within azd (that's ok as 3rd party lib is MIT, so we just need to aknowledge)

**Comment by rajeshkamal5050** [comment]
@jongio @vhvb1989 @ellismg 
Moving to On Deck since PR #91 has dependencies on another library requiring changes too.

**Comment by savannahostrowski** [comment]
@Austinauth  I think you'd also been playing around with some similar UX treatments here. I'll move this to Gallium per our convo about UX enhancements post-Build.

**Comment by Austinauth** [comment]
@savannahostrowski Yep - have been exploring this. There are a few options we can look at further. Last time I we discussed this (over 6 months ago) it didn't seem like there was much we could do to modify the list select component.

**Comment by rajeshkamal5050** [comment]
@savannahostrowski @vhvb1989 moving this to backlog. Please pull this back in if the dependency we had on the external library is fixed or has the feature.

**Comment by kristenwomack** [comment]
Hey @ccbarragan and @SophCarp bringing this issue back up because we still have template display improvements we could make to improve the template list usability.

<img width="2466" height="1256" alt="Image" src="https://github.com/user-attachments/assets/82d21872-3f3a-4877-a674-1afec1d6494c" />

<img width="1198" height="528" alt="Image" src="https://github.com/user-attachments/assets/5c1fe911-0dbe-40dc-95fd-ef31f4e3ffe0" />

**Comment by rajeshkamal5050** [comment]
Similar to the other issue - https://github.com/Azure/azure-dev/issues/6175

We have made updates to `azd template list` and `azd init` to show the galleries to further help with filtering and searching.
